Sprint-1/implement/max.js

```diff
@@ -1,4 +1,23 @@
 function findMax(elements) {
+  const numericValues = elements.filter(
+    (element) => typeof element === "number" && !isNaN(element)
+  );
+  let maxNumber = numericValues[0];
+  //console.log(numericValues);
+  if (
+    elements.length === 0 ||
+    elements.every((element) => typeof element != "number")
+  ) {
+    return "-Infinity";
```

Comment:
`Infinity` is a keyword in the JavaScript language. So you can return `-Infinity` (which is a value of type "number") instead of returning `"-Infinity"`.

Reply:
Amended. thanks